OpeningTimes.js

```diff
@@ -96,9 +81,33 @@ class OpeningTimes {
     return this._openingTimes[this._getDayName(moment)];
   }
 
-  _getOpenSessions(moment) {
+  _getOpeningTimesSessionForMoment(moment, daysLookAhead) {
+    // Get sessions for week (including yesterday for the cases where opening hours span midnight)
+    let returnValue;
+    for (let day = daysLookAhead - 1; day >= -1; day--) {
```

Comment:
The loop executes twice.

Once for today (i.e. day 0) and once for yesterday (i.e. day -1) to handle opening times which span midnight.

We could remove the loop and then call something like
 _getOpeningTimesSessionForMoment(moment, n) where n is 0 and -1 but I'm not sure what would be gained.